homeassistant/components/androidtv/media_player.py

```diff
@@ -192,12 +191,14 @@ async def _adb_exception_catcher(
                     err,
                 )
                 await self.aftv.adb_close()
+                # pylint: disable-next=protected-access
```

Comment:
My preference is usually to leave the normal `disable` (for the whole block) in-place, if I would need to add `disable-next` multiple times. It's just not worth it, especially for a low priority error like `protected-access`.

Reply:
I honestly don't agree.

If one uses protected access again, it means someone has to make an active choice. Either don't use protected or change the scope of disabling things.

Broadening it by default, may have side effects in future code.